pages/data-privacy.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Hero  from "../components/dataPrivacy/HeroSection";
+import styled from "styled-components";
+import { breakpoint } from "../frontend-config";
+
+const Headline = styled.h3`
+  font-size: 25px;
+  line-height: 29px;
+  margin-bottom: 12px;
+  margin-top: 30px;
+`;
+
+const Section = styled.div`
+  padding: 35px 75px 38px;
+  line-height: 1.5em;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 40px 75px 62px;
+  }
+`;
+
+export default function Home() {
+  return (
+    <>
+      <Hero />
```

Comment:
I mentioned in the 'Hero' file that we have a 'PageHeader' already, it just wraps around your title. You can see it in action on `pages/contribute` for reference.